.golangci.yml

```diff
@@ -251,15 +238,6 @@ issues:
     - typecheck
     text: import (".+") is a program, not an importable package
     path: ^tools\.go$
-  # TODO(sbueringer) Ignore ifshort false positive: https://github.com/esimonov/ifshort/issues/23
-  - linters:
-    - ifshort
-    text:  "variable 'isDeleteNodeAllowed' is only used in the if-statement.*"
-    path: ^internal/controllers/machine/machine_controller\.go$
-  - linters:
-    - ifshort
-    text: "variable 'kcpMachinesWithErrors' is only used in the if-statement.*"
-    path: ^controlplane/kubeadm/internal/workload_cluster_conditions\.go$
   # We don't care about defer in for loops in test files.
   - linters:
     - gocritic
```

Comment:
Nit: could also delete as a minor cleanup:

```
 skip-dirs:
  - third_party
```
 https://github.com/kubernetes-sigs/cluster-api/blob/3f6f56fb556cd8855b85dd9e878d5e373fc26ede/.golangci.yml#L277

Reply:
Addressed in: https://github.com/kubernetes-sigs/cluster-api/pull/7122/files